src/components/YieldPools/ListItem.tsx

```diff
@@ -671,6 +687,20 @@ const ListItem = ({ farm }: ListItemProps) => {
                   </Trans>
                 </GetLP>
               </Link>
+
+              {!!FARMING_POOLS_CHAIN_STAKING_LINK[farm.id.toLowerCase()] && (
+                <ExternalLink href={`${FARMING_POOLS_CHAIN_STAKING_LINK[farm.id.toLowerCase()]}`}>
+                  <GetLP style={{ display: '-webkit-inline-box' }}>
+                    <Trans>Earn More!</Trans>{' '}
+                    <InfoHelper
+                      color={theme.primary}
+                      text={t`Earn even more rewards by staking your LP Tokens here!`}
```

Comment:
Please re-run yarn start to update translation file

Reply:
done